lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -221,6 +243,9 @@ class MailboxDashBoardController extends ReloadableController {
 
   void _getUserProfile() async {
     userProfile.value = sessionCurrent != null ? UserProfile(sessionCurrent!.username.value) : null;
+    if (kIsWeb && userProfile.value != null) {
+      _handleEmailCache();
+    }
```

Comment:
It is so weird. Why `_handleEmailCache` should be handle in `_getUserProfile`? Hard to understand?

Reply:
we need userProfile != null to get composer cache. Because of that I _handleComposerCache in _getUserProfile

Reply:
I don't care much. I need it readable and make sense